packages/core/useFetch/index.test.ts

```diff
@@ -0,0 +1,24 @@
+import { useFetch } from '.'
+
+describe('useFetch', () => {
+  const urls = {
+    get: 'https://httpbin.org/get',
+    post: 'https://httpbin.org/get',
+    error: 'https://httpbin.org/status/500',
+  }
+
+  test('basic get request', async() => {
+    const { statusCode } = useFetch(urls.get)
+
+    /**
```

Comment:
Would like some input on the best way to go about writing these tests in this situation

Reply:
The comment starts on line 13

Reply:
We can make the `execute()` async. And they can also use `when`

```ts
await when(isFinished)
```

Reply:
Sounds good, I'll make those changes and create some basic tests